packages/aragon-wrapper/src/index.js

```diff
@@ -1346,3 +1426,6 @@ export { resolve as ensResolve } from './ens'
 
 // Re-export the Aragon RPC providers
 export { providers } from '@aragon/messenger'
+
+// Re-export the AddressIdentityProvider abstract base class
+export { AddressIdentityProvider } from './identity'
```

Comment:
Not super sure where exactly this should live, feels really unnecessary to make it a separate package.